charts/scalardl-audit/charts/envoy

```diff
@@ -0,0 +1 @@
+../../envoy
```

Comment:
This file should not work if Charts are installed using the following steps, so don't commit it.

```
helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
helm install scalar-labs/scalardl
```

Reply:
I apologize for the inconvenience. I'll delete it.